src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import './UserInfo.scss';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user.name}
+  </a>
```

Comment:
destructuring everywhere)

```suggestion
 const {
    name,
    email,
  } = user;

  <a className="UserInfo" href={`mailto:${email}`}>
    {name}
  </a>
```